### DISCUSSION TITLE
JSON schema multipleOf not working

### DISCUSSION THREAD

**Discussion by qhuang66** [json-schema-org/community#621] [Q&A]
I have a schema using the multipleOf, but seems not working, I define a number with multipleOf 0.000000000000001(15 decimal places) in the schema, but if I use a number with less decimal places, like 1.23, it will fail the validation saying 1,23 is not a multiple of 1e-15.

**Comment by gregsdennis** [comment]
It likely depends on the tool you're using, but my guess is that it can't resolve numbers that precisely.
